### PR TITLE
Allowing us to set directories to unmanaged

### PR DESCRIPTION
**Pull request by danzilio**

### CODE REVIEW

File: features/update.feature [hunk 1]
```diff
@@ -98,6 +98,86 @@ Feature: update
...
+    And a file named "modules/puppetlabs-apache/spec_helper.rb" with:
+      """
+      This is a fake spec_helper!
+      """
+    When I run `msync update --offline`
+    Then the output should contain:
+      """
+      Not managing spec/spec_helper.rb in puppetlabs-apache
+      """
+    And the exit status should be 0
```
Comment (igalic) [inline features/update.feature]: https://travis-ci.org/puppet-community/modulesync/jobs/97000306#L511-L518 this test is failing on ruby 2.2

**Comment by igalic** [comment]
how do we get this new version out?

**Comment by danzilio** [comment]
I think we need to modulesync modulesync

**Comment by igalic** [comment]
…

**Pull request merged**